portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,15 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+function addRandomFact() {
```

Comment:
Nit: I know this name comes from `addGreeting`, but the use of "add" seems a bit misleading (this function isn't "adding" something to something else). It's returning something - maybe `getRandomFact()`?